<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="won50">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50won</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Money">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="won100">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100won</string>
   </property>
  </widget>
  <widget class="QPushButton" name="won500">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500won</string>
   </property>
  </widget>
  <widget class="QPushButton" name="won10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10won</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coffee100">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tea150">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="milk200">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
